src/video_core/vulkan_common/vulkan_device.cpp

```diff
@@ -368,6 +369,15 @@ Device::Device(VkInstance instance_, vk::PhysicalDevice physical_, VkSurfaceKHR
     };
     SetNext(next, demote);
 
+    if (driver_id == VK_DRIVER_ID_AMD_PROPRIETARY || driver_id == VK_DRIVER_ID_AMD_OPEN_SOURCE) {
+        const u32 version = (properties.driverVersion << 3) >> 3;
```

Comment:
```suggestion
        const u32 version = properties.driverVersion;
```
The bitwise shifts masks the driver variant